sdk/spring/azure-spring-boot-test-aad-b2c/src/test/java/com/azure/test/aad/b2c/selenium/AADB2CIT.java

```diff
@@ -55,12 +59,22 @@ public void testProfileEdit() throws InterruptedException {
 
     @Test
     public void testLogOut() throws InterruptedException {
-        AADB2CSeleniumITHelper AADB2CSeleniumITHelper = new AADB2CSeleniumITHelper(DumbApp.class,
+        aadB2CSeleniumITHelper = new AADB2CSeleniumITHelper(DumbApp.class,
             Collections.emptyMap());
-        String signInButtonText = AADB2CSeleniumITHelper.logoutAndGetSignInButtonText();
+        aadB2CSeleniumITHelper.signIn(AADB2CTestUtils.AAD_B2C_SIGN_UP_OR_SIGN_IN);
+        String signInButtonText = aadB2CSeleniumITHelper.logoutAndGetSignInButtonText();
         Assert.assertEquals("Sign in", signInButtonText);
     }
 
+    @After
+    public void quitDriver() {
+        try {
+            aadB2CSeleniumITHelper.quitDriver();
```

Comment:
Maybe we could handle the exception within the aadB2CSeleniumITHelper.quitDriver() method.